tests/docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 version: '2.4'
 
 services:
-  app:
+  test:
+    image: lametro
     restart: "no"
     command: pytest -sxv
+    environment:
+      DJANGO_SETTINGS_MODULE: 'councilmatic.settings'
```

Comment:
Is is possible to update this in `pytest.ini` instead of here and in the Travis config? (Put another way, is there any reason to leave the old value in `pytest.ini`?)

Reply:
factored in out.